templates/webpack.typescript.js

```diff
@@ -158,7 +158,7 @@ module.exports = env => {
                 },
 
                 {
-                    test: /-page\.ts$/,
+                    test: /-page\.[jt]s$/,
```

Comment:
I think it would be more readable and extensible like this `test: /-page\.(js|ts)$/,`